readme.md

```
@@ -0,0 +1,254 @@
+├── frontend/             # React Router v7 + Vite
+│   ├── app/
+│   │   ├── routes/       # Page components (file-based routing)
+│   │   ├── features/     # Self-contained feature modules
+│   │   ├── components/   # Reusable UI components
+│   │   ├── contexts/     # React Context (auth)
+│   │   └── services/     # API clients
+│   └── public/           # Static assets
+│
+├── shared/               # TypeScript types & Zod schemas
+│   ├── src/
+│   │   ├── schemas/      # Zod validation schemas
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)